.github/workflows/ci.yml

```diff
@@ -0,0 +1,49 @@
+name: CI
+
+on: push
+
+jobs:
+  specs:
+    runs-on: ubuntu-latest
+
+    services:
+      mysql:
+        image: mysql:5.7
+        ports:
+          - 3306:3306
+        env:
+          MYSQL_ALLOW_EMPTY_PASSWORD: "yes"
+        options: >-
+          --health-cmd "mysqladmin ping -h localhost"
+          --health-interval 5s
+          --health-timeout 3s
+
+    strategy:
+      matrix:
+        ruby-version:
+          - 2.4
+          - 2.5
+          - 2.6
+          - 2.7
```

Comment:
3.0 is out too

Reply:
It is, but the official Docker image isn’t ready yet 😞